src/rpcrawtransaction.cpp

```diff
@@ -529,8 +541,18 @@ Value signrawtransaction(const Array& params, bool fHelp)
 
             "\nResult:\n"
             "{\n"
-            "  \"hex\": \"value\",   (string) The raw transaction with signature(s) (hex-encoded string)\n"
-            "  \"complete\": n       (numeric) if transaction has a complete set of signature (0 if not)\n"
+            "  \"hex\" : \"value\",   (string) The hex-encoded raw transaction with signature(s)\n"
+            "  \"complete\" : b,    (boolean) Whether the transaction has a complete set of signatures\n"
+            "  \"errors\" : [       (json array of objects) Script verification errors (if there are any)\n"
+            "    {\n"
+            "      \"txid\" : \"hash\",       (string) The transaction hash\n"
+            "      \"vout\" : n,            (numeric) The output number\n"
+            "      \"scriptSig\" : \"hex\",   (string) The hex-encoded script\n"
+            "      \"sequence\" : n,        (numeric) Script sequence number\n"
```

Comment:
Sequence number only has an effect on finality of transactions, which shouldn't usually be a cause of non-acceptance (if it is, it'll just be reported as non-final...).